### ISSUE TITLE
`org.junit.Assume.assumeTrue` is too powerful

### ISSUE THREAD

**Issue by Voileexperiments** [Codewars/codewars-runner-cli#602]
Simply do `import org.junit.Assume` then `Assume.assumeTrue(false);` completely destroys test assertion and allows user to solve almost every kata. Unless user manually counts how many expected and actual values are equal (which I have done in [become immortal](https://www.codewars.com/kata/become-immortal) for other reasons), this is unstoppable.

**Comment by Voileexperiments** [comment]
Nah, it doesn't terminate the test early, it instead makes `assert.Equals` and such broken because both `false` and `true` are accepted by them internally.

Things like `assert.Fail` will still work though.

**Comment by kazk** [comment]
Hm, I misunderstood this then.

> If called with an expression evaluating to `false`, the test will halt and be ignored.
>
> https://junit.org/junit4/javadoc/4.12/org/junit/Assume.html#assumeTrue(boolean)

So it's more like overriding assertion methods? Either way, I'm not doing anything about it on the runner. Maybe Codewars can flag the solution that uses it automatically.
